src/Tribe/Assets.php

```diff
@@ -774,4 +775,19 @@ public function get_js_dynamic_data() {
 		return $data;
 	}
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$css_template" missing

<sub>hash: a51411238bd952ceabe75f32aec2b82e</sub>